al-jazari/roster.json

```diff
@@ -42,5 +42,9 @@
       "name": "Tom Hacker",
       "github": "TommyHacker"
     }
```

Comment:
Can you please add a comma after the closing curly brace on line 44? @nathanolds